Vagrantfile

```diff
@@ -111,6 +111,7 @@ Vagrant.configure('2') do |config|
   config.vm.provision 'drupalvm', type: provisioner do |ansible|
     ansible.compatibility_mode = '2.0'
     ansible.playbook = playbook
+    ansible.install_mode = 'pip3'
```

Comment:
Ahhhh... in my previous readings I was thinking this was changing something inside the VM with the pip install that may happen as a result of Drupal VM's playbooks.

This is changing the Vagrantfile so that installs of _ansible_ in the VM if you need it installed inside will work correctly.

Thanks for catching this!

Reply:
Thank you!  Provisioning works 👍